apps/documents/urls.py

```diff
@@ -0,0 +1,21 @@
+from django.urls import path
+
+from apps.documents import views
+
+app_name = "documents"
+
+urlpatterns = [
+    path("<slug:tab_name>", views.RepositoryHome.as_view(), name="repositories"),
+    # Files
+    path("files/", views.FileListView.as_view(), name="files_list"),
+    path("files/<int:id>/details", views.FileDetails.as_view(), name="file_details"),
+    path("files/upload", views.upload_files, name="upload_files"),
```

Comment:
nit: move this up one so that the `files/<int:id>` urls are grouped. Same below for `new_collection`